pkg/rungroup/rungroup.go

```diff
@@ -59,23 +60,36 @@ func (g *Group) Run() error {
 		"actor_count", len(g.actors),
 	)
 
-	errors := make(chan actorError, len(g.actors))
+	actorErrors := make(chan actorError, len(g.actors))
 	for _, a := range g.actors {
-		go func(a rungroupActor) {
+		a := a
+		gowrapper.Go(g.slogger, func() {
 			g.slogger.Log(context.TODO(), slog.LevelDebug,
 				"starting actor",
 				"actor", a.name,
 			)
 			err := a.execute()
-			errors <- actorError{
+			actorErrors <- actorError{
 				errorSourceName: a.name,
 				err:             err,
 			}
-		}(a)
+		}, func(r any) {
+			g.slogger.Log(context.TODO(), slog.LevelInfo,
+				"shutting down after actor panic",
+				"actor", a.name,
+			)
```

Comment:
If this is a common pattern, do you think it's even worth logging in the wrapper?

Reply:
Yes, I do -- since panics are a big deal, I figured worth it to err on the side of over-logging and always logging in the wrapper.

Reply:
Okay. My gut says the other way, but I'll roll with "too many logs" 😆